src/wiki/help-pages/Data-storage.md

```diff
@@ -0,0 +1,26 @@
+---
+eleventyNavigation:
+  key: Data Storage
+  parent: Help pages
+--- 
+# Data Storage
```

Comment:
Help pages are for the help buttons in the launcher. Let's put it in `getting-started` for now. (you need to move the file there as well)

```suggestion
---
eleventyNavigation:
  key: Data Locationn
  parent: Getting Started
--- 
# Data Location
```